props/prop_base.h

```diff
@@ -1571,11 +1571,15 @@ class PropBase : CommandParser, protected Looper, protected SaberBase, public Mo
       return true;
     }
     if (!strcmp(cmd, "n") || (!strcmp(cmd, "next") && arg && (!strcmp(arg, "preset") || !strcmp(arg, "pre")))) {
-      next_preset();
+      SaberBase::IsOn() ? next_preset_fast() : next_preset();
```

Comment:
I'm trying to decide if I like this.
I mean, wouldn't you still want to hear the font sound when switching presets when it's on?

Reply:
We currently do not play font.wav when fast switching presets. 
I do not remember if that was ever a discussion, but personally I think that's appropriate. 
Announcing the font is a "when off" thing in my mind like a volume or blade length utility.
Quotes are available for any "while on" dialogue.

Anyway, this change is nothing more than aligning the behavior of driving the board with serial commands to how it actually works on saber.

Reply:
It makes sense that next_preset_fast / prev_preset_fast doesn't play the font name, i didn't mean that should change.  I'm just wondering if calling the "fast" functions when on is actually the right thing to do?

Reply:
If the real saber is on, and you do next preset, it does the fast version. Unless the prop doesn't support that, then I guess it does nothing? I might be guilty of assuming the behavior, and that's what I based the suggestion on.
I run my prop, and it does fast when saber is on, so for *me* it's mirroring the real world saber.
It's totally fine if this goes away, I just thought it was an alignment.

Reply:
> If the real saber is on, and you do next preset, it does the fast version.

Does it? That's not a given to me. 
Does all the props do this? If so, then I guess this makes sense.
The "saber" prop doesn't really have a way to change preset while on.

I'm asking because I really don't know what is the right behavior.
Calling two different functions costs a little extra memory, but probably not enough to matter.

Reply:
> Does all the props do this?

No, apparently only BC and FETT263 props do.
It's fine. I'll withdraw this.